### DISCUSSION TITLE
Virtual Display in Windows 11 VM

### DISCUSSION THREAD

**Discussion by Brett-Gladman** [MolotovCherry/virtual-display-rs#73] [General]
I was testing out your virtual display driver in a Windows 11 eval VM (a VMWare VM) and I've noticed that the mouse pointer does not appear to be in the correct position on the primary monitor.
When I click, the click is actually occurring maybe 100 to 200 pixels to the right of the displayed mouse point.
If I disable the virtual monitors, the mouse pointer and its click appears to return to the correct position.

I tested the driver on an actual Windows 11 machine, it worked exactly as expected. Excellent work by the way.

Has anyone else experienced this?

**Comment by MolotovCherry** [comment]
Interesting. I have no idea what the VM could be doing 🤔

**Comment by MolotovCherry** [comment]
Update. In #81 the fix was made clear. This will be fixed in time
